lab2/devops-js-app_remote-server.conf

```diff
@@ -0,0 +1,70 @@
+proxy_buffering off;
+
+upstream front_end_app {
+    server localhost:4200;
+}
+
+upstream rest_api_server {
+    server localhost:3000; #provide port of your back-end's app
+}
+
+server {
+    listen 5555;
+    location / {
+        proxy_pass http://front_end_app;
+        proxy_set_header Upgrade $http_upgrade;
+        proxy_set_header Connection $http_connection;
+        proxy_set_header Host $host;
+    }
+
+    location /api/ {
+        proxy_pass http://rest_api_server/;
+        proxy_http_version 1.1;
+        proxy_set_header Host $host;
+        proxy_set_header X-NginX-Proxy true;
+        proxy_set_header X-Real-IP $remote_addr;
+        proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
+        proxy_set_header X-Forwarded-Proto $scheme;
+    }
+}
+
+server {
+    listen 8080;
+    server_name static-app.net;
+    root /Users/Dzianis_Pasiukou1/Documents/projects/shop-angular-cloudfront/dist/app;
```

Comment:
This config is not reusable due to path to the app.
Pls consider storing you app in `/usr/local/var/www/` directory


same as "root html/..." for nginx